terraform/modules/shared/main.tf

```diff
@@ -459,22 +459,83 @@ resource "azurerm_container_app_environment_storage" "tabulation_storage" {
 
 #### Kubernetes Service ####
 
+resource "azurerm_virtual_network" "aks_vnet" {
+  name                = "phdi-${terraform.workspace}-vnet"
+  resource_group_name = var.resource_group_name
+  address_space       = ["10.30.0.0/16"]
+  location            = var.location
+
+  subnet {
+    name           = "phdi-${terraform.workspace}-aks_subnet"
+    address_prefix = "10.30.1.0/24"
+  }
+}
+
+resource "azurerm_network_security_group" "aks_nsg" {
+  name                = "phdi-${terraform.workspace}-nsg"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+
+  security_rule {
+    name                       = "AzureDataFactoryInbound"
+    priority                   = 100
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "*"
+    source_port_range          = "*"
+    destination_port_range     = "*"
+    source_address_prefix      = "DataFactory"
+    destination_address_prefix = "*"
+  }
+  security_rule {
+    name                       = "AzureDataFactoryOutbound"
+    priority                   = 100
+    direction                  = "Outbound"
+    access                     = "Allow"
+    protocol                   = "*"
+    source_port_range          = "*"
+    destination_port_range     = "*"
+    source_address_prefix      = "*"
+    destination_address_prefix = "DataFactory"
+  }
+  security_rule {
```

Comment:
This rule looks like a copy of the one above - could you remove it?

Reply:
yes, not sure why thats there...